### PR TITLE
Refactor/breadcrumbs

### PR DESCRIPTION
**Pull request by AdamChrimes**
## Description

Refactoring the breadcrumb to meet the component criteria checklist, along with psuedocode tests, visual testing and examples of the component.

## Breaking changes

Breadcrumbs is now known as Breadcrumb, this doesn't affect the code but how the verbal naming of it.

## Component checklist

- [x] SCSS
- [x] SCSS lint
- [x] HTML template
- [x] HTML validate & lint
- [x] Nunjucks macro
- [x] A standalone example
- [x] README/Documentation
- [x] Pseudocode tests
- [x] Visual tests 
- [x] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] Print stylesheet considered
- [x] CHANGELOG

### CODE REVIEW

File: packages/assets/icons/icon-chevron-left.svg [hunk 1]
```diff
@@ -1,4 +1,3 @@
 <svg class="nhsuk-icon nhsuk-icon__chevron-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
   <path d="M8.5 12c0-.3.1-.5.3-.7l5-5c.4-.4 1-.4 1.4 0s.4 1 0 1.4L10.9 12l4.3 4.3c.4.4.4 1 0 1.4s-1 .4-1.4 0l-5-5c-.2-.2-.3-.4-.3-.7z"/>
```
Comment (mcheung-nhs) [inline packages/assets/icons/icon-chevron-left.svg]: Need to close the `<path>` with `</path>` for consistency and as can been seen in the templates.

File: packages/assets/icons/icon-chevron-right.svg [hunk 2]
```diff
@@ -1,4 +1,4 @@
 <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
   <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"/>
```
Comment (mcheung-nhs) [inline packages/assets/icons/icon-chevron-right.svg]: Need to close the `<path>` with `</path>` for consistency and as can been seen in the templates.

File: packages/assets/icons/icon-chevron-left.svg (hunk 1 above)
Comment (AdamChrimes) [reply packages/assets/icons/icon-chevron-left.svg]: Resolved

File: packages/assets/icons/icon-chevron-right.svg (hunk 2 above)
Comment (AdamChrimes) [reply packages/assets/icons/icon-chevron-right.svg]: Resolved

**Pull request merged**